src/Rules/Api/ApiInterfaceExtendsRule.php

```diff
@@ -66,10 +65,7 @@ private function checkName(Scope $scope, Node\Name $name): array
 			'https://github.com/phpstan/phpstan/discussions',
 		))->build();
 
-		if (in_array($extendedInterfaceReflection->getName(), [
-			Type::class,
-			ReflectionProvider::class,
-		], true)) {
+		if (in_array($extendedInterfaceReflection->getName(), BcUncoveredInterface::CLASSES, true)) {
```

Comment:
I am re-using the same constant as in `ApiClassImplementsRule`.. thats how I understand the todo